sdk/digitaltwins/Azure.DigitalTwins.Core/src/Models/IDigitalTwinRelationship.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.DigitalTwins.Core
+{
+    /// <summary>
+    /// The service-defined fields of a digital twin relationship.
+    /// </summary>
+    public interface IDigitalTwinRelationship
```

Comment:
Introducing an interface for relationship.

Reply:
What are your thoughts on adding constants here that tell users what the name of each json property is? I'm considering doing so in Java

Reply:
```csharp
const DigitalTwinRelationshipIdJsonPropertyName = "$relationshipId"
```

Reply:
It's important that we convey these json property names somewhere, after all

Reply:
I don't think a constant helps them any more than a comment, because they can't refer to the comment in an attribute, can they?

Reply:
Also, with our current runtime, we can't put default impl on interfaces yet. That is too new of a .NET feature.

Reply:
Found constants can be in an attribute. I've added that info in comments. Will leave it this way for GA, and we can always add constants later if they are requested.